### PR TITLE
Move benchmark driver doc into README

### PR DESCRIPTION
**Pull request by m57lyra**
Used pandocs to convert rst file
Added converted README.md file to presto-benchmark-driver
Removed rst file 
Removed mention of rst file in installation.rst

### CODE REVIEW

**Comment by mosabua** [comment]
Looks like the md file renders ok in github 
https://github.com/m57lyra/presto/tree/benchmark-driver-readme/presto-benchmark-driver

File: presto-benchmark-driver/README.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+Benchmark Driver
+================
+
+The benchmark driver can be used to measure the performance of queries
+in a Presto cluster. We use it to continuously measure the performance
+of the master branch.
+
+Download :maven_download:[benchmark-driver]{.title-ref}, rename it to
```
Comment (mosabua) [inline presto-benchmark-driver/README.md:8]: This definitely needs to be changed to be a valid URL.

File: presto-benchmark-driver/README.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+------
+
+The benchmark driver measures the wall time, total CPU time used by all
+Presto processes and the CPU time used by the query. For each timing,
+the driver reports median, mean and standard deviation of the query
+runs. The difference between process and query CPU times is the query
+overhead, which is normally from garbage collections. The following is
+the output from the `file_formats` suite above:
+
+``` {.none}
```
Comment (mosabua) [inline presto-benchmark-driver/README.md:57]: We might need to change this table to be more markdownish as well

File: presto-benchmark-driver/README.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+output.
+
+Another way to create additional output columns is by adding tags to the
+SQL files. For example, the following SQL file declares two tags,
+`projection` and `filter`:
+
+```
+projection=true
+filter=false
+=================
```
Comment (mosabua) [inline presto-benchmark-driver/README.md:86]: I am not sure this should be there .. is this not a remnant from the rst file?

File: presto-benchmark-driver/README.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+## Output
+
+The benchmark driver measures the wall time, total CPU time used by all
+Presto processes and the CPU time used by the query. For each timing,
+the driver reports median, mean and standard deviation of the query
+runs. The difference between process and query CPU times is the query
+overhead, which is normally from garbage collections. The following is
+the output from the `file_formats` suite above:
+
+```
```
Comment (mosabua) [inline presto-benchmark-driver/README.md:57]: Please convert this into a markdown table instead of using a source code block.

**Review by mosabua — APPROVED** [review]
Everything moves over cleanly. Good job.

**Comment by mosabua** [comment]
I am requesting review from @electrum to get this merged.

**Comment by mosabua** [comment]
I also checked what the markdown renders in github and it looks good.

**Review by mosabua — CHANGES_REQUESTED** [review]
Download still needs to be fixed

File: presto-benchmark-driver/README.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
+# Benchmark Driver
+
+The benchmark driver can be used to measure the performance of queries
+in a Presto cluster. We use it to continuously measure the performance
+of the master branch.
+
+ Download the [Maven benchmark driver plugin][maven_download] for the same
```
Comment (mosabua) [inline presto-benchmark-driver/README.md:7]: This needs to point to Central for download..

Comment (mosabua) [reply presto-benchmark-driver/README.md:7]: https://repo.maven.apache.org/maven2/io/prestosql/presto-verifier/

Comment (mosabua) [reply presto-benchmark-driver/README.md:7]: https://repo.maven.apache.org/maven2/io/prestosql/presto-benchmark-driver/

**Review by electrum — APPROVED** [review]

**Pull request merged**